<robot name="ADEPT_VIPER_S650">

	<link name="Adept_Viper_s650_Link1">
		<visual>
			<origin xyz="0 0 0" rpy="0 0 0"/>
			<geometry>
				<mesh filename="package://adept_common/meshes/visual/STL/binary/Adept_Viper_650_Link1.stl"/>
			</geometry>
			<material name="White">
				<color rgba="255 255 255 1.0"/>
			</material>
		</visual>
		<collision>
			<origin xyz="0 0 0" rpy="0 0 0"/>
			<geometry>
				<mesh filename="package://adept_common/meshes/collision/STL/binary/Adept_Viper_650_Link1.stl"/>
			</geometry>
			<material name="Blue">
				<color rgba="0 0 255 1.0"/>
			</material>
		</collision>
	</link>
	<link name="Adept_Viper_s650_Link2">
		<visual>
			<origin xyz="0 0 0" rpy="0 0 0"/>
			<geometry>
				<mesh filename="package://adept_common/meshes/visual/STL/binary/Adept_Viper_650_Link2.stl"/>
			</geometry>
			<material name="Blue">
				<color rgba="0 0 255 1.0"/>
			</material>
		</visual>
		<collision>
			<origin xyz="0 0 0" rpy="0 0 0"/>
			<geometry>
				<mesh filename="package://adept_common/meshes/collision/STL/binary/Adept_Viper_650_Link2.stl"/>
			</geometry>
			<material name="Blue">
				<color rgba="0 0 255 1.0"/>
			</material>
		</collision>
	</link>	
	<link name="Adept_Viper_s650_Link3">
		<visual>
			<origin xyz="0 0 0" rpy="0 0 0"/>
			<geometry>
				<mesh filename="package://adept_common/meshes/visual/STL/binary/Adept_Viper_650_Link3.stl"/>
			</geometry>
			<material name="Blue">
				<color rgba="0 0 255 1.0"/>
			</material>
		</visual>
		<collision>
			<origin xyz="0 0 0" rpy="0 0 0"/>
			<geometry>
				<mesh filename="package://adept_common/meshes/collision/STL/binary/Adept_Viper_650_Link3.stl"/>
			</geometry>
			<material name="Blue">
				<color rgba="0 0 255 1.0"/>
			</material>
		</collision>
	</link>
	<link name="Adept_Viper_s650_Link4">
		<visual>
			<origin xyz="0 0 0" rpy="0 0 0"/>
			<geometry>
				<mesh filename="package://adept_common/meshes/visual/STL/binary/Adept_Viper_650_Link4.stl"/>
			</geometry>
			<material name="Blue">
				<color rgba="0 0 255 1.0"/>
			</material>
		</visual>
		<collision>
			<origin xyz="0 0 0" rpy="0 0 0"/>
			<geometry>
				<mesh filename="package://adept_common/meshes/collision/STL/binary/Adept_Viper_650_Link4.stl"/>
			</geometry>
			<material name="Blue">
				<color rgba="0 0 255 1.0"/>
			</material>
		</collision>
	</link>
	<link name="Adept_Viper_s650_Link5">
		<visual>
			<origin xyz="0 0 0" rpy="0 0 0"/>
			<geometry>
				<mesh filename="package://adept_common/meshes/visual/STL/binary/Adept_Viper_650_Link5.stl"/>
			</geometry>
			<material name="Blue">
				<color rgba="0 0 255 1.0"/>
			</material>
		</visual>
		<collision>
			<origin xyz="0 0 0" rpy="0 0 0"/>
			<geometry>
				<mesh filename="package://adept_common/meshes/collision/STL/binary/Adept_Viper_650_Link5.stl"/>
			</geometry>
			<material name="Blue">
				<color rgba="0 0 255 1.0"/>
			</material>
		</collision>
	</link>
	<link name="Adept_Viper_s650_Link6">
		<visual>
			<origin xyz="0 0 0" rpy="0 0 0"/>
			<geometry>
				<mesh filename="package://adept_common/meshes/visual/STL/binary/Adept_Viper_650_Link6.stl"/>
			</geometry>
			<material name="Blue">
				<color rgba="0 0 255 1.0"/>
			</material>
		</visual>
		<collision>
			<origin xyz="0 0 0" rpy="0 0 0"/>
			<geometry>
				<mesh filename="package://adept_common/meshes/collision/STL/binary/Adept_Viper_650_Link6.stl"/>
			</geometry>
			<material name="Blue">
				<color rgba="0 0 255 1.0"/>
			</material>
		</collision>
	</link>
	<joint name="joint_1" type="revolute">
		<origin xyz="0 0 0.203"/>
		<parent link="Adept_Viper_s650_Link1"/>
		<child link="Adept_Viper_s650_Link2"/>
		<axis xyz="0 0 1"/>
		<limit effort="100" velocity="1.0" lower="-2.9670" upper="2.9670"/>
	</joint>
	<joint name="joint_2" type="revolute">
		<origin xyz="0.074 0 0.18795"/>
		<parent link="Adept_Viper_s650_Link2"/>
		<child link="Adept_Viper_s650_Link3"/>
		<axis xyz="0 1 0"/>
		<limit effort="100" velocity="1.0" lower="-3.31613" upper="0.7854"/>
	</joint>
	<joint name="joint_3" type="revolute">
		<origin xyz="0 0 0.270"/>
		<parent link="Adept_Viper_s650_Link3"/>
		<child link="Adept_Viper_s650_Link4"/>
		<axis xyz="0 1 0"/>
		<limit effort="100" velocity="1.0" lower="-0.50615" upper="4.468053"/>
	</joint>
	<joint name="joint_4" type="revolute">
		<origin xyz="0.295 0 0.090"/>
		<parent link="Adept_Viper_s650_Link4"/>
		<child link="Adept_Viper_s650_Link5"/>
		<axis xyz="1 0 0"/>
		<limit effort="100" velocity="1.0" lower="-3.31613" upper="3.316133"/>
	</joint>
	<joint name="joint_5" type="revolute">
		<origin xyz="0 0 0"/>
		<parent link="Adept_Viper_s650_Link5"/>
		<child link="Adept_Viper_s650_Link6"/>
		<axis xyz="0 1 0"/>
		<limit effort="100" velocity="1.0" lower="-2.0944" upper="2.0944"/>
	</joint>	
</robot>
